apps/prairielearn/elements/pl-matching/pl-matching.py

```diff
@@ -343,6 +344,7 @@ def render(element_html, data):
             "counter_type": counter_type,
             "no_counters": no_counters,
             "options_placement": options_placement.value,
+            "editable": editable,
```

Comment:
Nit, you can inline this.

```suggestion
            "editable": data["editable"],
```